Docs/source/usage/parameters.rst

```diff
@@ -1229,17 +1229,19 @@ Numerics and algorithms
     computational medium, respectively. The default values are the corresponding values
     in vacuum.
 
-* ``interpolation.nox``, ``interpolation.noy``, ``interpolation.noz`` (`1`, `2`, or `3` ; default: 1)
-    The order of the shape factors for the macroparticles, for the 3 dimensions of space.
-    Lower-order shape factors result in faster simulations, but more noisy results,
+* ``interpolation.shape_factors_order`` (`1`, `2`, or `3`)
+    The order of the shape factors for the macro-particles along all spatial directions.
```

Comment:
```suggestion
    The order of the shape factors for the macro-particles along all spatial directions: `1` for linear, `2` for quadratic and `3` for cubic.
```
Even if it is evident, I would write it explicitly.

Reply:
as far as `warpx.shape_factors_order` vs `interpolation.shape_factors_order` is of concern, I don't have specific preferences. I think that the latter is close to the previous version, so I think that we can keep the solution that you have adopted, for the moment. More in general, I don't know why certain options have `warpx` as a prefix and others do not (except for those that have `amrex` as prefix, of course!).

Reply:
Thank you for your comments, Luca. I cleaned this up in https://github.com/ECP-WarpX/WarpX/pull/1934/commits/e016246eee23bfacbfd58f0a3b73ea691f733d63.

About the prefix of our input parameters, I guess the idea was to make them as self-explanatory as possible, while keeping them as short as possible. For instance, before this PR we had `interpolation.nox` as well as `psatd.nox`, which needed to be distinguished. One could have used the `warpx` prefix anyways, such as `warpx.interpolation_nox` and `warpx.psatd_nox`, to make a clear distinction between all `amrex` parameters and all `warpx` parameters, but I guess people maybe wanted to keep the names as short as possible. Might also be related to how the same parameters were named originally in Warp, to make it easier to compare the two codes.